helmfile/01-applications.yaml

```
@@ -148,6 +148,17 @@ releases:
   values:
   - values/metrics-server.yaml.gotmpl
 
+# calico-accountant
+- name: calico-accountant
+  namespace: kube-system
+  labels:
+    app: calico-accountant
+  chart: ./charts/calico-accountant
+  version: 0.1.0
```

Maybe not install this if the dashboard is disabled? Or create another way of enable/disable this maybe?
Or do we always want this?

```suggestion
  version: 0.1.0
  installed: {{ .Values.dashboards.networkpolicy.enabled }}
```

---

I think that sounds like a good way to enable/disable.

---

I could add this, but whether or not the dashboard is enabled is currently only set in the [grafana-ops values](https://github.com/elastisys/compliantkubernetes-apps/blob/f33d38574d0e787d27ee91b3af5a32d0881eeaa5/helmfile/charts/grafana-ops/values.yaml), not in the config (same with all other grafana dashboards, as far as I can tell?)

So I'd have to add a value to the config for this. Would it make sense to add that just for this dashboard specifically?

---

Yes, that is correct! Hmm, yea then I think we can leave this as it is.

---

I would be happy to be able to disable the exporter while i don't care as much about the grafana dashboard.
It does not have to be a 1-1 mapping imo, but others might disagree. I mean I'm fairly sure we have grafana dashboards from the kubernetes mixins that are empty becuse we dont collect all metrics.

Thoughts?

---

Yea, now that you mention it. Since we might install this on a cluster that uses some other CNI than calico.

---

And as you say, it would be nice if you also could disable the dashboard although it's not super important. That is probably something for a later issue, because then I think we need to do it for all dashboards

---

Sound good to just add enable/disable for only calico-accountant

---

I could add enable/disable just for calico-accountant, but is there any reason not to let that setting also extend to enabling/disabling the NetworkPolicy dashboard?

---

Maybe in some multi-cluster setup you could perhaps want the dashboard in sc to just look at metrics from workload clusters. So I don't think they need to be connected.